spec/lcm/integration/shared_contexts_for_lcm_specs.rb

```diff
@@ -0,0 +1,264 @@
+require_relative 'support/configuration_helper'
+require_relative 'support/project_helper'
+require_relative 'support/connection_helper'
+require_relative 'support/lcm_helper'
+require 'gooddata_datawarehouse'
+
+def create_suffix
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)